### PR TITLE
WIP:  Fix engine import test

### LINKED ISSUE #44: kratos tests are not properly running
From https://travis-ci.org/simphony/simphony-framework/builds/69049405:

```
haas simkratos -v

----------------------------------------------------------------------

Ran 0 tests in 0.000s
```

### LINKED ISSUE #45: tests/test_engine_import.py does not report failure
If some of the engines are not installed, the test does not report a failure

### PR DESCRIPTION
**Pull request by nathanfranklin**
This PR:
* Fixes how extension names are tested and updates the extension names for JYULB and Openfoam wrappers  (fixes #45) 
* Added kratos to make step in travis.yml  (fixes #44)

### CODE REVIEW

**Comment by nathanfranklin** [comment]
@itziakos .  I took a stab at #45 .  Can you see if this looks alright.  (Note that this PR is set to merge these commits to upgrade-openfoam-0.1.3)

**Comment by nathanfranklin** [comment]
@itziakos ,  hold off on looking at this.   its failing cause the build/tests for Travis-CI is failing as it is taking longer than 50 minutes.  i will look at addressing that on monday.

**Comment by itziakos** [comment]
@nathanfranklin, this is a good idea.

File: .travis.yml [hunk 1]
```diff
@@ -6,16 +6,22 @@ virtualenv:
   system_site_packages: true
 env:
   - SIMPHONYENV=~/simphony
+  - TEST_SUITE=test-simphony
```
Comment (itziakos) [inline .travis.yml]: The matrix is not setup properly. let make fix it.

Comment (nathanfranklin) [inline .travis.yml]: @itziakos , should it become something like the following?:
```
env:
  global:
      - SIMPHONYENV=~/simphony
  matrix:
      -  TEST_SUITE=test-simphony
      -  TEST_SUITE=test-jyulb
      -  TEST_SUITE=test-lammps
...
```

Comment (itziakos) [inline .travis.yml]: yes, I just pushed the changes. I am also checking some more speed-ups

File: Makefile [hunk 2]
```diff
@@ -35,8 +35,16 @@ help:
 	@echo "  simphony-mayavi   to build and install the simphony-mayavi plugin"
 	@echo "  simphony-openfoam to build and install the simphony-openfoam plugin"
 	@echo "  simphony-jyu-lb   to build and install the simphony-jyu-lb plugin"
+	@echo "  test-simphony     run the tests for the simphony library"
+	@echo "  test-kratos       run the tests for the simphony-kratos plugin"
+	@echo "  test-lammps   	   run the tests for the simphony-lammps plugin"
+	@echo "  test-numerrin     run the tests for the simphony-numerrin plugin"
+	@echo "  test-mayavi       run the tests for the simphony-mayavi plugin"
+	@echo "  test-openfoam     run the tests for the simphony-openfoam plugin"
+	@echo "  test-jyu-lb       to build and install the simphony-jyu-lb plugin"
```
Comment (nathanfranklin) [inline Makefile]: "run the tests for"

**Comment by nathanfranklin** [comment]
@itziakos , thanks!  Left one comment, but otherwise looks good to me.

(Note that this PR is set to merge these commits to upgrade-openfoam-0.1.3 but should be now actually merge to master)

**Comment by itziakos** [comment]
closing to make a new pr against master

**Pull request closed without merging**